core/source/Http.mint

```diff
@@ -168,7 +170,9 @@ module Http {
     |> Http.send()
   */
   fun formDataBody (body : FormData, request : Http.Request) : Http.Request {
-    { request | body = `#{body}` }
+    { request |
+      body = `#{body}`,
+      headers = [...request.headers, { key = "content-type", value = "multipart/form-data" }] }
```

Comment:
This is specifically recommended **against** in the [MDN article](https://developer.mozilla.org/en-US/docs/Web/API/FormData/Using_FormData_Objects).

Reply:
Oh right so the browser sets it by default.

Reply:
Cool, I removed it 👍